provider/filestore.go

```diff
@@ -114,19 +114,21 @@ func (store AzureFileStore) AsAzureStore() *AzureFileStore {
 }
 
 func (store AzureFileStore) PathWithPrefix(path string, remote bool) string {
+	pathContainsAzureBlobPrefix := strings.HasPrefix(path, azureBlobPrefix)
```

Comment:
Can you add comments in these functions for what the expected path should look like and/or add tests that enforce it

Reply:
Is this referenced somewhere?

Reply:
I added tests for `PathWithPrefix` and the `pathContainsAzureBlobPrefix` is referenced in the conditional statement below.